edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.Http/Adapters/HttpsExtensionConnectionAdapter.cs

```diff
@@ -0,0 +1,305 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+namespace Microsoft.Azure.Devices.Edge.Hub.Http.Adapters
+{
+    using System;
+    using System.Collections.Generic;
+    using System.IO;
+    using System.Linq;
+    using System.Net.Security;
+    using System.Security.Cryptography.X509Certificates;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Microsoft.AspNetCore.Http.Features;
+    using Microsoft.AspNetCore.Server.Kestrel.Core.Adapter.Internal;
+    using Microsoft.AspNetCore.Server.Kestrel.Https;
+    using Microsoft.Azure.Devices.Edge.Hub.Http.Middleware;
+    using Microsoft.Azure.Devices.Edge.Util;
+    using Microsoft.Extensions.Logging;
+
+    // https://github.com/aspnet/HttpAbstractions/issues/808
+    public class HttpsExtensionConnectionAdapter : IConnectionAdapter
+    {
```

Comment:
Looks like you have modified this file slightly.. in that case we should align it with EdgeHub semantics. Alternatively, I am okay this file being completely unmodified..

Reply:
This file has not been modified apart from log messages and certificate and a few Edge utilities and Edge hub conventions.